fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -284,88 +204,31 @@ format_classpath() {
 # ==========================================================================
 
 memory_options() {
-  echo "$(calc_init_memory) $(calc_max_memory)"
+  echo "$(set_max_memory)"
   return
 }
 
-# Check for memory options and set max heap size if needed
-calc_max_memory() {
+# Check for memory options and set max heap size as percentage of RAM, if needed
+set_max_memory() {
+  local MaxRAMPercentage=""
+  local MinRAMPercentage=""
+
   # Check whether -Xmx is already given in JAVA_OPTIONS
   if echo "${JAVA_OPTIONS:-}" | grep -q -- "-Xmx"; then
     return
   fi
-
-  if [ -z "${CONTAINER_MAX_MEMORY:-}" ]; then
-    return
+  
+  if ! (echo "${JAVA_OPTIONS:-}" | grep -q -- "-XX:MaxRAMPercentage"); then
+    MaxRAMPercentage="-XX:MaxRAMPercentage=85.0"
   fi
-
-  # Check for the 'real memory size' and calculate Xmx from the ratio
-  if [ -n "${JAVA_MAX_MEM_RATIO:-}" ]; then
```

Comment:
Why removing support for this variable ? It could be trivially translated to `MinRAMPercentage`  / `MasRAMPercentage`.

Reply:
Having this in addition to MinRAMPercentage and MaxRAMPercentage does not make sense and adds to confusion. Keeping things simple and to the point is always good.